common/configtx/update.go

```diff
@@ -32,9 +32,8 @@ func (vi *ValidatorImpl) verifyReadSet(readSet map[string]comparable) error {
 func computeDeltaSet(readSet, writeSet map[string]comparable) map[string]comparable {
 	result := make(map[string]comparable)
 	for key, value := range writeSet {
-		readVal, ok := readSet[key]
 
-		if ok && readVal.version() == value.version() {
+		if readVal, ok := readSet[key]; ok && readVal.version() == value.version() {
```

Comment:
Although it is less lines of code, my personal opinion is that the change makes it less readable. Regardless of whether it is a little better or a little worse, what is the motivation for making what is essentially a no-op change? If there is value then it's fine, I'm just concerned with polluting the commit history and diverging branches if there is not a clear benefit.